tool/plugin/lib/plugin.dart

```diff
@@ -97,12 +120,17 @@ class AntBuildCommand extends ProductCommand {
 
   AntBuildCommand(this.runner);
 
-  String get name => 'build';
+  String get name => 'abuild';
```

Comment:
is `abuild` intended?

Reply:
For now, to distinguish from 'build'. It will be removed eventually.

Reply:
👍